_layouts/cask.html

```diff
@@ -84,6 +84,64 @@
 </table>
 {%- endif %}
 
+{%- if c.variations.size > 1 -%}
```

Comment:
I know there was a comment on this, but I think this actually should be `if c.variations.size > 0`. The `variations` item only includes things that differ from the main information, so if there are really no variations to list then `c.variations` will be `{}`.

For example, take [`amethyst`](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/a/amethyst.rb). On El Capitan, it has a different version so the generated hash is:

```json
      "version": "0.20.0",
      ...
      "variations": {
        "el_capitan": {
          "url": "https://ianyh.com/amethyst/versions/Amethyst-0.10.1.zip",
          "url_specs": {
          },
          "version": "0.10.1",
          "sha256": "9fd1ac2cfb8159b2945a4482046ee6d365353df617f4edbabc4e8cadc448c1e7"
        }
      },
```

Right now, on the [cask page](https://formulae.brew.sh/cask/amethyst#default), it doesn't show any variatinos

Reply:
Whoops, my bad, thanks @Rylan12 and fixing in https://github.com/Homebrew/formulae.brew.sh/pull/1010.